docs/website/versioned_docs/version-2.5.0/getting-started/configure.md

```diff
@@ -96,7 +96,6 @@ Unsetting a preference key sets it to an empty value in the preference file. odo
 | Preference         | Description                                                                    | Default                |
 |--------------------|--------------------------------------------------------------------------------|------------------------|
 | UpdateNotification | Control whether a notification to update odo is shown                          | True                   |
-| NamePrefix         | Set a default name prefix for an odo resource (component, storage, etc)        | Current directory name |
```

Comment:
But this doc is for v2 docs and still makes sense for v2, no? `NamePrefix` seems to still be there in v2 code: 
https://github.com/redhat-developer/odo/blob/v2/pkg/preference/implem.go#L26-L27

I checked that current doc for v3 does not mention `NamePrefix`; it was removed in [#5585](https://github.com/redhat-developer/odo/pull/5585/files#diff-23a7570482e5623ef8af8fcf3dea364e8d59daaa4ea0aaa06ac56dfbad90d752)